doc/releases/release-notes-3.6.rst

```diff
@@ -601,36 +548,34 @@ Drivers and Sensors
     * :c:func:`regulator_set_active_discharge`
     * :c:func:`regulator_get_active_discharge`
     * :c:func:`regulator_list_current_limit`
```

Comment:
Needs extra whitespace to avoid being mashed into a single line

```suggestion

    * :c:func:`regulator_set_active_discharge`
    * :c:func:`regulator_get_active_discharge`
    * :c:func:`regulator_list_current_limit`

```